### PR TITLE
Rename instances of resume to profile <PD-1495>

### PR DESCRIPTION
**Pull request by tdphillips**
A quick grep shows several more instances of "resume" but those are in job descriptions in microsite test feeds.

Tests pass.

### CODE REVIEW

File: templates/mydashboard/candidate_information.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
         <table><tr>
         <td>{% get_gravatar the_user '100' %}</td>
         <td><div id="candidate-completion">
-            <b><div >Resume is <span id="highlight" class="initial-highlight">{{ the_user.profile_completion }}% complete</span></div></b>
+            <b><div >Profile is <span id="highlight" class="initial-highlight">{{ the_user.profile_completion }}% complete</span></div></b>:w
```
Comment (aspidites) [inline templates/mydashboard/candidate_information.html]: > :w 

Vim motion?

**Comment by tdphillips** [comment]
@fezick This contains changes to the privacy policy and terms.

File: templates/mydashboard/candidate_information.html (hunk 1 above)
Comment (tdphillips) [inline templates/mydashboard/candidate_information.html]: You saw nothing.

File: templates/terms.html [hunk 2]
```diff
@@ -24,7 +24,7 @@
...
     <li>You grant My.jobs a nonexclusive, irrevocable, worldwide, perpetual, unlimited, assignable, sublicenseable, fully paid up and royalty-free right to us to copy, prepare derivative works of, improve, distribute, publish, remove, retain, add, process, analyze, use and commercialize, in any way now known or in the future discovered, any information you provide, directly or indirectly to My.jobs, including, but not limited to, any user generated content, ideas, concepts, techniques and/or data to the services, you submit to My.jobs, without any further consent, notice and/or compensation to you or to any third parties. Any information you submit to us is at your own risk of loss. </li>
     <li>By providing information to us, you represent and warrant that you are entitled to submit the information and that the information is accurate, not confidential, and not in violation of any contractual restrictions or other third party rights. </li>
-    <li>It is your responsibility to keep your My.jobs resume information accurate and updated.</li>
+    <li>It is your responsibility to keep your My.jobs profile information accurate and updated.</li>
```
Comment (aspidites) [inline templates/terms.html]: Totally a legit change, but given this is technically changing terms, do we need to pass this off to the legal team for approval first?

File: templates/terms.html [hunk 3]
```diff
@@ -79,7 +79,7 @@
...
     <li>You may also opt-in to receive non-account emails and Saved Search from <a href="{% url 'edit_account' %}">Account Settings</a>.</li>
     <li>My.jobs collects, uses, stores, and discloses information about you in keeping with our Privacy Policy.</li>
-    <li>Carefully read our full <a href="{% url 'privacy' %}">Privacy Policy</a> before using My.jobs as it is hereby incorporated into this Agreement by reference, and governs our treatment of any information, including personally identifiable information you submit to us. Please note that certain information, statements, data, and content (such as your resume photo) which you may submit to My.jobs are likely to, reveal your gender, ethnic origin, nationality, age, and/or other personal information about you. You acknowledge that your submission of any information, statements, data, and content to us is voluntary on your part and that My.jobs may process such information, within the terms of the <a href="{% url 'privacy' %}">Privacy Policy</a>.</li>
```
Comment (aspidites) [inline templates/terms.html]: Which begs the question- Aren't these terms a bit out of date, considering they refer to things which can't actually be done anymore (such as uploading a profile photo)?

Comment (tdphillips) [inline templates/terms.html]: The closest we got to photos was using Gravatar. We've never allowed anyone to upload anything until fairly recently.

**Comment by fezick** [comment]
Changing resume to profile in TOS is fine. Don't change anything else. Capitalization doesn't really matter, so leave it lowercase.

**Comment by JLMcLaugh** [comment]
@tdphillips @mklauber There's one test failing, probably due to Jenkins changes. It may be easier to change the test to use unicode explicitly than figure out what's going on in Jenkins.

**Pull request merged**